django/contrib/auth/models.py

```diff
@@ -357,6 +357,14 @@ def get_short_name(self):
         "Returns the short name for the user."
         return self.first_name
 
+    def get_hash_value(self, timestamp):
+        """Returns the timestamped hash value for purposes of generating
```

Comment:
pydoc can show the docstring of the parent method if the subclass has no docstring in Python 3.6 (or could be 3.5). I think we can remove this one to avoid duplicate docstrings.